src/shared/components/palette/collection/write-collection-field.component.ts

```diff
@@ -103,6 +112,35 @@ export class WriteCollectionFieldComponent extends AbstractFieldWriteComponent i
     return index ? `${this.caseField.label} ${displayIndex}` : this.caseField.label;
   }
 
+  isNotAuthorisedToCreate() {
+    if (this.isExpanded) {
+      return false;
+    }
+    return !this.profile.user.idam.roles.find(role => !!this.caseField.acls.find( acl => acl.role === role && acl.create === true));
+  }
+
+  isNotAuthorisedToUpdate(index: number) {
+    if (this.isExpanded) {
+      return false;
+    }
+    let id = false;
+    if (this.formArray.at(index)) {
+      id = this.formArray.at(index).get('id').value;
+    }
+    return !!id && !this.profile.user.idam.roles.find(role => !!this.caseField.acls.find( acl => acl.role === role && acl.update === true));
```

Comment:
Can we extract `acl.role === role && acl.update === true` to a separate method like `hasUpdateAccess`.